index.js

```diff
@@ -1,4 +1,7 @@
 import Toast from './src';
 
 export { default as BaseToast } from './src/components/base';
+export { default as SuccessToast } from './src/components/success';
+export { default as ErrorToast } from './src/components/error';
+export { default as InfoToast } from './src/components/info';
```

Comment:
If we are to do this, the type definition file would need to be updated as well
https://github.com/calintamas/react-native-toast-message/blob/master/index.d.ts